certbot/auth_handler.py

```diff
@@ -33,14 +33,18 @@ class AuthHandler(object):
         and values are :class:`acme.messages.AuthorizationResource`
     :ivar list achalls: DV challenges in the form of
```

Comment:
Yeah it's still true. `ivar` stands for instance variable and `achalls` is still a variable of the `AuthHandler` class.